Loop/Models/LoopSettings+Loop.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 import LoopCore
+import LoopKit
```

Comment:
What makes this import required now?

Reply:
Not required, it got removed in a prior PR but I forgot to remove it in this once. Will fix